framework/src/engine/endpoint/system.ts

```diff
@@ -95,6 +96,7 @@ export class SystemEndpoint {
 			transaction: transactionSchema,
 			event: eventSchema,
 			standardEvent: standardEventDataSchema,
+			ccm: ccmSchema,
```

Comment:
CCM is the concept of the interoperability. It should be exposed from the interoperability module